test/Main.purs

```diff
@@ -84,6 +86,8 @@ main = do
   quickCheck     $ 4 >?  3
   quickCheckFail $ 4 <=? 3
 
+  randomSample (MGen.chooseFloat ((-1.7976931348623157e+308)) (1.7976931348623157e+308)) >>= assert <<< all isFinite
```

Comment:
Can you add a log statement just before this line so that if this fails, it will be a little clearer as to why?